.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+
+language: bash
+
+services:
+  - docker
+
+install:
+  - docker build -t quay.io/${TRAVIS_REPO_SLUG:0:6}/${TRAVIS_REPO_SLUG:14}:${TRAVIS_COMMIT:0:7} .
```

Comment:
Is there an advantage to using substring operations on `TRAVIS_REPO_SLUG` here? If the repo name or organization name changed, it would break, and in the `deploy:` section we have hardcoded the content of `TRAVIS_REPO_SLUG`.